packages/webapp/pages/settings/organization/[orgId]/members.tsx

```diff
@@ -128,23 +179,31 @@ const OrganizationMembersItem = ({
 
   const blocked = contentPreference?.status === ContentPreferenceStatus.Blocked;
 
+  /* TODO: fetch real value */
```

Comment:
@omBratteng this still needs to be build on API?

Reply:
Yes